mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -496,8 +504,8 @@ private JobExecution verifyJobExecution(JobExecution jobExecution) {
     if (jobExecution.getStatus() == JobExecution.Status.ERROR || jobExecution.getStatus() == COMMITTED
     || jobExecution.getStatus() == JobExecution.Status.CANCELLED) {
       String msg = String.format("JobExecution with status '%s' cannot be forcibly completed", jobExecution.getStatus());
-      LOGGER.error(msg);
-      throw new BadRequestException(msg);
+      LOGGER.info(msg);
+      throw new JobUpdateException(msg);
```

Comment:
Maybe it makes sense to rename this exception something like this: JobUpdateDuplicateException (when we trying to update status that was already updated previously).
It will be more clear to answer question why we logging it as info, and why for it we return succeeded future